frontend/src/components/ScheduledEventsRunHistoryTableCard/utils/index.ts

```diff
@@ -0,0 +1,24 @@
+import * as localForage from 'localforage';
+import { api, localForageKeys } from '../../../utils';
+import axios from 'axios';
+
+const BASEPATH = api.routes.VIEW_SCHEDULED_EVENTS;
+
+export const fetchRunHistory = async (eventId: number): Promise<any> => {
+  const token = await localForage.getItem<string>(localForageKeys.API_KEY);
+  const headers = {
+    'Content-Type': 'application/json',
+    Authorization: `token ${token}`,
+  };
+
+  return await axios(`${BASEPATH}${eventId}/run_instances/`, { headers });
```

Comment:
Use a constant with a placeholder that your can then replace with string.replace